### PR TITLE
Add Delete complete

### PR DESCRIPTION
**Pull request by i-max-xi**
✔️  Implemented add function
✔️  Implemented remove function
✔️  Implemented edit function
✔️  Local Storage tracks all changes

### CODE REVIEW

File: src/add.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
+export class TaskObj {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+
+  static taskArr = [];
+
```
Comment (ZeenatLawal) [inline src/add.js:9]: - [ ] The todo items are been added to the localStorage but are not reflecting on the page when refreshed, kindly fix this.

File: src/add.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+  OK.id = 'ok';
+  newHead.appendChild(OK);
+  const OKbtn = document.querySelector('#ok');
+
+  OKbtn.addEventListener('click', () => {
+    content.innerText = newContent.value;
+    newContent.remove();
+    OK.remove();
+  });
+};
```
Comment (ZeenatLawal) [inline src/add.js]: - [ ] The changes made to an item using the edit function does not save to the local storage, kindly fix this.

**Review by ZeenatLawal — CHANGES_REQUESTED** [review]
Hi @i-max-xi, 

Good job so far!:clap: :clap:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎊 🎊 
- GitHub flow is followed. ✔️
- PR has a title and summary. ✔️


## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by dasileker — APPROVED** [review]
## **Hi @i-max-xi,** 

## **Congratulations! 🎉 Project Approved ✅**
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 
 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**